core/ui/EditTemplate.tid

```diff
@@ -8,13 +8,26 @@ title: $:/core/ui/EditTemplate
 <$action-deletetiddler $tiddler=<<newFieldValueTiddler>>/>
 <$action-sendmessage $message="tm-save-tiddler"/>
 \end
+\define focus-selector()
+[data-tiddler-title="$(cssEscapedTitle)$"].tc-tiddler-frame input
```

Comment:
This selector will target the first input element in the tiddler frame. Perhaps we should respect the new autofocus setting we use for focussing new tiddlers?

Reply:
(Plus the same point to the two further examples below)

Reply:
That makes sense @Jermolene. I'm trying to do so but I'm having a hard time targeting the framed text-editor. Maybe you know a trick for doing so?